tests/StoringGroceries.tex

```diff
@@ -16,6 +16,7 @@ \subsection{Focus}
 		\item \textbf{Cupboard:} The cupboard has 5 shelves between 0.30m and 1.80m from the ground and contains several objects (See \ref{rule:scenario_objects}).
 		\begin{itemize}
 		 	\item \textbf{Door:} The cupboard has a single door, which is closed initially.
+		 	This door covers half of the objects, e.g. the left/right half of the closet, as indicated by the hatched area in the image.
```

Comment:
I would rather say ```This door encloses some of the objects, covering up to one half of the cupboard (e.g. the left/right...```

This is because we don't know beforehand how he cupboard will look like.